pandas/core/window/expanding.py

```diff
@@ -135,12 +140,12 @@ def count(self, **kwargs):
     @Appender(_shared_docs["apply"])
     def apply(
         self,
-        func,
+        func: Callable,
         raw: bool = False,
         engine: Optional[str] = None,
         engine_kwargs: Optional[Dict[str, bool]] = None,
-        args=None,
-        kwargs=None,
+        args: Optional[Tuple] = None,
+        kwargs: Optional[Dict] = None,
```

Comment:
we will eventually want to use disallow_any_generic as we increase strictness checks xref #30539

so for kwargs could probably use `Dict[str, Any]` and for args, `Tuple[Any, ...]`

can you also add type parameters to Callable.

Reply:
Sounds good. Done

Reply:
Thanks @mroeschke lgtm